packages/coil-extension/src/content/services/RunContentHandler.ts

```diff
@@ -16,6 +16,12 @@ interface GetPageData {
   }
 }
 
+interface HyperlinkElement extends Element {
+  href: string
+}
+
+const CHANNEL_ID_REGEX = /https?:\/\/(www\.)?youtube\.com\/channel\//i
```

Comment:
This appears unused ?

Reply:
Yeah, was in an earlier attempt that I forgot to remove.